packages/charting/src/components/StackedBarChart/StackedBarChart.base.tsx

```diff
@@ -148,12 +153,14 @@ export class StackedBarChartBase extends React.Component<IStackedBarChartProps,
       }
       value = (pointData / total) * 100;
       startingPoint.push(prevPosition);
-      const isSelected = this.state.legendSelected === point.legend!;
       const styles = this.props.styles;
+      let shouldHighlight = true;
+      if (this.state.isLegendHovered || this.state.isLegendSelected) {
+        shouldHighlight = this.state.selectedLegendTitle === point.legend ? true : false;
```

Comment:
Nit: unnecessary ternary operator (see above)

Reply:
Removed it.